troposphere/static/js/components/admin/AtmosphereUser.jsx

```diff
@@ -60,7 +61,7 @@ export default React.createClass({
             <td style={{ border: "none" }} 
                 className="user-name"
             >
-                {user.get("username")}
+                {user.get("username")} <Emulate username={user.get("username")} />
```

Comment:
Unless we have a motivation for this regarding mobile-friendly, I was vote against the right alignment.